includes/wikia/services/TemplateClassificationService.class.php

```diff
@@ -77,10 +78,9 @@ public function getType( $wikiId, $pageId ) {
 	public function getDetails( $wikiId, $pageId ) {
 		$templateDetails = [];
 
-		$details = $this->getApiClient()->getTemplateDetails( $wikiId, $pageId );
+		$providers = $this->getApiClient()->getTemplateDetails( $wikiId, $pageId );
```

Comment:
My thinking is that the class name itself ```TemplateClassification``` tells you on what data you operate and for what this class is responsible. Do we need to repeat it in method naming?